ops/ansible/roles/bfd-pipeline/defaults/main.yml

```diff
@@ -11,9 +11,12 @@ data_pipeline_new_relic_metric_host: 'https://gov-metric-api.newrelic.com'
 data_pipeline_new_relic_metric_path: '/metric/v1'
 data_pipeline_ccw_rif_job_enabled: true
 
-rif_thread_multiple: 3
-rif_job_batch_size: 7
-rif_job_queue_size_multiple: 5
+rif_thread_multiple: 25
+rif_job_batch_size: 100
+rif_job_queue_size_multiple: 10
+rif_thread_multiple_claims: 25
+rif_job_batch_size_claims: 100
+rif_job_queue_size_multiple_claims: 10
```

Comment:
Not immediately understanding here; wasn't the point of separating these settings so that we could have _different_ values for the claims and non-claims settings? It seems like in all the locations here we've bumped the values but they are the same for claims and non-claims.